pom.xml

```diff
@@ -80,6 +80,19 @@
 
     <build>
         <plugins>
+            <plugin>
+                <!-- Release Plugin Configuration -->
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
+                <configuration>
+                    <!-- Prepare -->
+                    <tagNameFormat>@{project.version}</tagNameFormat>
+                    <pushChanges>false</pushChanges>
```

Comment:
Personal opinion, but IMO it's better to use properties so they can be overridden from the command line.